### PR TITLE
Fix description for multimedia

### PR DESCRIPTION
**Pull request by hsgwon**
### Bugs Fixed ###
- Fix typo in media-contnet.md, media-controller.md, media-playback.md, media-recording.md, raw-audio.md, stream-recorder.md
- Add missing privilege description in media-controller.md.
- Remove Tizen.Xamarin.Forms.Extension.MediaView.

### CODE REVIEW

**Comment by hsgwon** [comment]
Please do not review yet. More commits will be uploaded.

File: docs/application/dotnet/guides/multimedia/media-controller.md [hunk 1]
```diff
@@ -23,10 +23,11 @@ The main media controller features include:
 
 To enable your application to use the media controller functionality:
 
-1.  To use the [Tizen.Multimedia.Remoting.MediaControlServer](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+1.  To use the [Tizen.Multimedia.Remoting.MediaController](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Remoting.MediaController.html) and [Tizen.Multimedia.Remoting.MediaControlServer](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```
Comment (sabeena21) [inline docs/application/dotnet/guides/multimedia/media-controller.md]: Question: In the privileges we have client and server why is Tizen.Multimedia.Remoting.MediaControlServer changed to Tizen.Multimedia.Remoting.MediaController any specific reason?

**Comment by mijin85cho** [comment]
Hello, @hsgwon.
Would you please check the comments?

File: docs/application/dotnet/guides/multimedia/media-playback.md [hunk 2]
```diff
@@ -250,14 +250,14 @@ To play a video file:
 
 2. To set the display on which the video is played, use the `Display` property of the `Tizen.Multimedia.Player` class.
 
-    For example, to set the display on a Xamarin-based application, first create an instance of the [Tizen.Xamarin.Forms.Extension.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Xamarin.Forms.Extension.MediaView.html) class, cast it to an instance of the [Tizen.Multimedia.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.MediaView.html) class, and finally set that instance as the `Display` property:
+    For example, to set the display on a Xamarin-based application, first create an instance of the custom renderer(ex. VideoView) based on VisualElementRenderer class, cast it to an instance of the [Tizen.Multimedia.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.MediaView.html) class, and finally set that instance as the `Display` property:
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/multimedia/media-playback.md]: @sabeena21 Shouldn't we change "ex." to "For example"?

File: docs/application/dotnet/guides/multimedia/media-playback.md [hunk 3]
```diff
@@ -250,14 +250,14 @@ To play a video file:
...
 
     ```
-    var mediaView = new Tizen.Xamarin.Forms.Extension.MediaView();
+    var mediaView = new VideoView();
 
     mediaView.NativeViewCreated += (s, e) =>
     {
-        player.Display = new Display((Tizen.Multimedia.MediaView)mediaView.NativeView);
+        player.Display = new Display((Tizen.Multimedia.MediaView)(s as VideoView).NativeView);
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/multimedia/media-playback.md]: Is it really "s as VideoView"?

File: docs/application/dotnet/guides/multimedia/media-recording.md [hunk 4]
```diff
@@ -190,18 +190,25 @@ To record audio:
...
-1.  Create an instance of the [Tizen.Xamarin.Forms.Extension.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Xamarin.Forms.Extension.MediaView.html) class, which provides a view of the camera on the screen.
+1.  Create an instance of the [Tizen.Multimedia.Camera](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Camera.html) class.
+    ```
+    var camera = new Camera(CameraDevice.Rear);
+    ```
 
-    Create an instance of the [Tizen.Multimedia.Camera](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Camera.html) class and set the display of the camera to the `NativeView` property of the `Tizen.Xamarin.Forms.Extension.MediaView` instance:
+2. To set the display on which the video is recorded, use the `Display` property of the `Tizen.Multimedia.Camera` class.
+
+    For example, to set the display on a Xamarin-based application, first create an instance of the custom renderer(ex. VideoView) based on VisualElementRenderer class, cast it to an instance of the [Tizen.Multimedia.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.MediaView.html) class, and finally set that instance as the `Display` property:
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/multimedia/media-recording.md]: @sabeena21 Shouldn't we change "ex." to "For example"?

File: docs/application/dotnet/guides/multimedia/media-content.md [hunk 5]
```diff
@@ -649,41 +650,44 @@ The following tables list the available media file information.
...
+| `StorageId`      | The storage type of the storage that the media is stored on |
 
 **Table: Audio metadata (only for audio files)**
 
 | Metadata name    | Description                              |
 |----------------|----------------------------------------|
 | `Album`          | Album information for the audio content  |
 | `Artist`         | Artist of the audio content              |
-| `Album Artist`   | Album artist of the audio content<br>The artist and album artist can be the same. |
+| `AlbumArtist`    | Album artist of the audio content<br>The artist and album artist can be the same. |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/guides/multimedia/media-content.md]: About the \<br\> tag, as we can't control the width of a table,
it is not bad to maintain \<br\> tag.

[the remark this answers is not in the document]
Comment (sabeena21) [reply docs/application/dotnet/guides/multimedia/media-content.md]: @mijin85cho, okay

File: docs/application/dotnet/guides/multimedia/media-playback.md [hunk 6]
```diff
@@ -218,7 +218,7 @@ To play an audio file:
 1.  After creating the player, specify the audio file to play by creating an instance of the [Tizen.Multimedia.MediaUriSource](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.MediaUriSource.html) class with the path to the audio file, and setting that as the `Source` property of the [Tizen.Multimedia.Player](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Player.html) class instance you want to play the file with:
 
     ```
-    player.Source = new MediaUriSource(audioPath);
+    player.SetSource(new MediaUriSource(audioPath);
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/multimedia/media-playback.md]: Add ')'. (change to player.SetSource(new MediaUriSource(audioPath));)

File: docs/application/dotnet/guides/multimedia/media-content.md [hunk 7]
```diff
@@ -107,14 +107,14 @@ To enable your application to use the media content functionality:
...
-    MediaDatabase.MediInfoUpdated += OnMediaInfoUpdated;
+    MediaDatabase.MediaInfoUpdated += OnMediaInfoUpdated;
     MediaDatabase.FolderUpdated += OnFolderUpdated;
     ```
 
 2. When you no longer want to receive notifications, deregister the event handlers:
 
     ```
-    MediDatabase.MediInfoUpdated -= OnMediaInfoUpdated;
+    MediDatabase.MediaInfoUpdated -= OnMediaInfoUpdated;
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/multimedia/media-content.md]: line117,118: Change MediDatabase to MediaDatabase.

File: docs/application/dotnet/guides/multimedia/media-content.md [hunk 8]
```diff
@@ -649,41 +650,44 @@ The following tables list the available media file information.
...
 | `Latitude`       | Latitude of the media content            |
 | `Altitude`       | Altitude of the media content            |
 | `Rating`         | Rating of the media content              |
-| `Favorite`       | Favorite status of the media content     |
+| `IsFavorite`     | Favorite status of the media content     |
 | `Title`          | Title of the media content               |
-| `Storage type`   | Storage type of the media content        |
+| `StorageId`      | Storage ID of the media content          |
+| `IsDrm`          | The media is DRM-protected or not        |
+| `StorageType`      | Storage ID of the media content        |
```
Comment (mijin80lee) [inline docs/application/dotnet/guides/multimedia/media-content.md]: line 672: Check the description. (change to The storage type of the storage that the media is stored on?)
line 673: StorageId was duplicated.

File: docs/application/dotnet/guides/multimedia/media-content.md [hunk 9]
```diff
@@ -693,27 +697,27 @@ The following tables list the available media file information.
 |---------------|----------------------------------------|
 | `Width`         | Width of the image                       |
 | `Height`        | Height of the image                      |
-| `Exposure time` | Exposure time of the image               |
-| `F-number`      | F-number of the image                    |
-| `ISO`           | ISO of the image                         |
+| `ExposureTime`  | Exposure time of the image               |
+| `FNumber`       | F-number of the image                    |
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-content.md]: The property name is FNumber. I'll change the description.

File: docs/application/dotnet/guides/multimedia/media-content.md [hunk 10]
```diff
@@ -693,27 +697,27 @@ The following tables list the available media file information.
...
+| `Iso`           | ISO of the image                         |
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-content.md]: The property name is ```Iso```. I'll change the description.

File: docs/application/dotnet/guides/multimedia/media-content.md [hunk 11]
```diff
@@ -649,41 +650,44 @@ The following tables list the available media file information.
...
 |----------------|----------------------------------------|
-| `Media ID`       | ID of the media content                  |
-| `File path`      | Path of the media content                |
-| `Display name`   | Display name of the media content        |
-| `Media type`     | Media type of the media content          |
-| `Mime type`      | MIME type of the media content           |
-| `Size`           | File size of the media content           |
-| `Added time`     | Time the media content was added to the database |
-| `Modified time`  | Last modification time of the media content |
+| `Id`             | ID of the media content                  |
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-content.md]: The property name is ```Id```. I'll change the description.

File: docs/application/dotnet/guides/multimedia/media-content.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-content.md]: As mijin said, I'll keep the current statement.

File: docs/application/dotnet/guides/multimedia/media-playback.md (hunk 3 above)
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-playback.md]: Yes. you can check this Tizen.Multimedia.Manual.Tests, support/CameraTestBase.cs

File: docs/application/dotnet/guides/multimedia/media-playback.md (hunk 2 above)
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-playback.md]: I'll fix it.

File: docs/application/dotnet/guides/multimedia/media-recording.md (hunk 4 above)
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-recording.md]: I'll fix it.

File: docs/application/dotnet/guides/multimedia/media-content.md (hunk 7 above)
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-content.md]: I'll fix it.

File: docs/application/dotnet/guides/multimedia/media-content.md (hunk 8 above)
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/media-content.md]: I'll fix it.

**Comment by mijin85cho** [comment]
Thanks for the review & update.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**